.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+## Overview
+In general, this project is only a documentation or onboarding hub.  Please be kind and respectful to each other, be it in GitHub, Slack, or of course real life.
+This project is intended to be open and accessible, for the community by the community.
+
+For our general contributing guidelines, please see the [wiki](https://github.com/ProvidenceGeeks/website-docs/wiki)
+
+## Documentation
```

Comment:
@kpisano 
Any other good ideas you have around how to best draft doc would be good here, and then I can add them in.  Of course we can continue to refine over time, so no rush to get all in on the first try.